css-overflow-4/Overview.bs

```diff
@@ -1140,6 +1163,44 @@ Fragmentation of Overflow: the 'continue' property</h3>
 	as the box would be a [=flex container=] rather than a [=block container=],
 	and thus the 'continue' property would not apply.
 
+<h4 id=line-clamp-containers>Line-clamp containers</h4>
+
+	A <dfn>line-clamp container</dfn> is an [=independent formatting context|independent=] [=block formatting context=] that
+	additionally follows the rules in this section. All line-clamp containers contain a
+	<dfn>clamp point</dfn>, which is one of the following positions inside it:
+
+	- A point immediately after a [=line box=] in the line-clamp container's [=block formatting
+	  context=].
+	- The end of the line-clamp container, if there are any boxes after the last [=inline formatting
+	  context=] in the line-clamp container's [=block formatting context=].
+
+	If the [=line-clamp container=]'s [=block formatting context root=] has a [=computed value=] of
+	'max-lines' other than ''max-lines/none'', then that property will determine the [=clamp point=].
+	Otherwise, the clamp point will be set to the last possible clamp point such that, for it and
+	all previous possible clamp points, the line-clamp container's [=automatic block size=] (as
+	determined below) is not greater than the [=block size=] the box would have if its automatic
+	block size were infinite; or if that is not the case for any clamp points, to the first clamp
+	point in the block formatting context.
```

Comment:
Editorial:
```suggestion
	If the [=line-clamp container=]'s [=block formatting context root=]
	has a [=computed value=] of 'max-lines' other than ''max-lines/none'',
	then that property will determine the [=clamp point=].
	Otherwise, the clamp point will be set to the last possible clamp point
	such that, for it and all previous possible clamp points,
	the line-clamp container's [=automatic block size=]
	(as determined below)
	is not greater than the [=block size=] the box would have
	if its automatic block size were infinite;
	or if that is not the case for any possible clamp points,
	to the first possible clamp point in the block formatting context.
```
See https://rhodesmill.org/brandon/2012/one-sentence-per-line/

I also added the word "possible" before "clamp point" in the last two lines.